internal/controller/pipelinerollout_controller_test.go

```diff
@@ -968,6 +977,9 @@ func Test_processExistingPipeline_PPND(t *testing.T) {
 			// the Reconcile() function does this, so we need to do it before calling reconcile() as well
 			rollout.Status.Init(rollout.Generation)
 
+			err = numaplaneClient.Create(ctx, rollout)
+			assert.NoError(t, err)
```

Comment:
This is interesting. So, it didn't work if the PipelineRollout didn't already exist on the cluster?

(I mean, I suppose in real life, the PipelineRollout _would_ already exist if the Pipeline already exists)

Reply:
yeah, if not exist the `update()` will not work as it is not created yet.